akka-cluster-sharding-typed/src/main/scala/akka/cluster/sharding/typed/internal/ClusterShardingImpl.scala

```diff
@@ -202,6 +204,12 @@ import akka.japi.function.{ Function ⇒ JFunction }
     new LeastShardAllocationStrategy(threshold, maxSimultaneousRebalance)
   }
 
+  override val shardState: ActorRef[ClusterShardingQuery] = {
+    import akka.actor.typed.scaladsl.adapter._
+    val behavior = ShardingState.behavior(untypedSharding)
+    untypedSystem.systemActorOf(PropsAdapter(behavior), "typed-shard-state")
```

Comment:
that's not the "normal" naming convention for actors, well, we might have a few with that convention but I think we mostly prefer `typedShardState`

Reply:
after reviewing more I agree that it's not worth creating separate for this